student.rb

```diff
@@ -1,17 +1,13 @@
-class Student
-  attr_reader :age, :phase
-  attr_accessor :name
+require_relative 'school'
+
+class Student < School
 
   def initialize(options = {})
     @phase = 1
     @age = options.fetch(:age, 0)
     @name = options.fetch(:name, "")
   end
```

Comment:
The `@age` and `@name` lines are the same in each class -- DRY it up by pulling them into a super class (`School`) and calling super from here.